platforms/documentation/docs/src/docs/userguide/best-practices/best_practices.adoc

```diff
@@ -48,6 +48,11 @@ The items in this guide are meant to be _brief_, _understandable_, and _actionab
 - <<best_practices_dependencies.adoc#best_practices_for_dependencies,Best Practices for Dependencies>>
 - <<best_practices_tasks.adoc#best_practices_for_tasks,Best Practices for Tasks>>
 
-== Further Reference
+== Recommendations
+
+- <<performance.adoc#performance_gradle, Improving the Performance of Gradle Builds>>
+- <<organizing_gradle_projects.adoc#structuring_recommendations,Structuring Gradle Builds>>
```

Comment:
This section has a "Use a `gradle.properties` file, which I think we should replace with a link to the best practice in that PR.  WDYT?

I think the other things here good candidates to migrate to BP items as well.  So the understanding would be that this link is temporary, since I think all the Structuring Recommendations become best practices eventually, right?

Reply:
I actually want to delete any redundancy altogether. Item has been deleted.